### DISCUSSION TITLE
Using Material UI + NextJS: I have a problem with Drawer

### DISCUSSION THREAD

**Discussion by SOLUTRAFFICDEVELOPERALPHA** [vercel/next.js#11378] [Help]
# Drawer Problem

I have a problem, i dont know why, but when I reload the page, the drawer styles popup

![image](https://user-images.githubusercontent.com/57099578/77694504-1b1f0d80-6f78-11ea-944b-b2d8a4e9e3fa.png)

At the start or reload the server there is no problem, but when i reload the page.

![image](https://user-images.githubusercontent.com/57099578/77694582-3f7aea00-6f78-11ea-8b2a-e948d1b83bce.png)

**Comment by jonahsnider** [comment]
Almost definitely a known Material-UI SSR CSS issue. Fix is documented [here](https://material-ui.com/styles/advanced/#next-js).

**Comment by Avi-Aravindh** [comment]
Hello, were you able to resolve the issue with the new example? Please confirm. Thanks
